src/Twilio/Recording.hs

```diff
@@ -63,3 +63,10 @@ instance Get1 RecordingSID Recording where
 -- | Get a 'Recording' by 'RecordingSID'.
 get :: MonadThrow m => RecordingSID -> TwilioT m Recording
 get = Resource.get
+
+instance Delete1 RecordingSID where
+  delete1 (getSID -> sid) = request parseJSONFromResponse =<< makeTwilioDELETERequest
+    ("/Recordings/" <> sid <> ".json")
+
+delete :: MonadThrow m => RecordingSID -> TwilioT m ()
+delete = Resource.delete
```

Comment:
I see the following warning in the Travis CI log output, which, because `-Werror` is defined, fails the build:

```
/home/travis/build/markandrus/twilio-haskell/src/Twilio/Recording.hs:72:1: Warning:
    Defined but not used: ‘Twilio.Recording.delete’
```

Can you add this to the exports list above? It should be something like the following (I sorted `get` and `delete` alphabetically, but I don't know if we follow a different pattern elsewhere in the source).

```hs
 module Twilio.Recording
   ( -- * Resource
     Recording(..)
   , Twilio.Recording.delete
   , Twilio.Recording.get
   ) where
```